### PR TITLE
Exercise 1 - Day1 completed

### PR DESCRIPTION
**Pull request by GOKULANANTH-MOHANASUNDARAM**

### CODE REVIEW

File: Day1/PrimeNumber.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+			copy/=10;
+		}
+		if(reverse == n) {
+			System.out.println(n + " is a palindrome");
+		}
+		else {
+			System.out.println(n + " is not a palindrome");
+		}
+	}
+
```
Comment (AbishekDuraisamy) [inline Day1/PrimeNumber.java:93]: Handle the case where a number is none of these any formats (armstrong, palindrome, perfect).. and print that result as well

File: Day1/PrimeNumber.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import java.util.Scanner;
+
+public class PrimeNumber {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter the nth value: ");
+		int n = sc.nextInt();
+		Prime p = new Prime(n);
+		p.finishprime();
```
Comment (AbishekDuraisamy) [inline Day1/PrimeNumber.java:11]: naming should follow camelCase for variables and methods

**Review by AbishekDuraisamy — COMMENTED** [review]
Some general Reviews that applies to all the files here:

- Few snippets need proper Indentation 
- Give meaningful names to the variables.
- Give verbs for method/function names. (function names should represent an action)
- use camelCase for variables/methods and PascalCase for Classes. 
- give proper spacing between operators and operands for better readability

File: Day1/PrimeNumber.java (hunk 1 above)
Comment (GOKULANANTH-MOHANASUNDARAM) [reply Day1/PrimeNumber.java:93]: Brother I had done this as an if-else statement

**Pull request closed without merging**